daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/ImmArray.scala

```diff
@@ -124,62 +118,68 @@ final class ImmArray[+A] private (
     * but rather returning an empty array. If you want Seq's behavior,
     * use `relaxedSlice`.
     */
-  def strictSlice(from: Int, until: Int): ImmArray[A] = {
-    if (from < 0 || from >= length || until < 0 || until > length) {
+  def strictSlice(from: Int, until: Int): ImmArray[A] =
+    if (from < 0 || from >= length || until < 0 || until > length)
```

Comment:
If `from == length` is forbidden, shouldn't `until == 0` be forbidden as well?

Reply:
If you wish I can the change in a separate PR.

Reply:
Makes sense. Thanks.